manifests/0000_30_cluster-api_00_credentials-request.yaml

```diff
@@ -70,14 +70,77 @@ metadata:
+    - Microsoft.ApiManagement/service/workspaces/tags/read
+    - Microsoft.ApiManagement/service/workspaces/tags/write
+    - Microsoft.Authorization/roleAssignments/read
+    - Microsoft.Authorization/roleAssignments/write
+    - Microsoft.Compute/availabilitySets/delete
+    - Microsoft.Compute/availabilitySets/write
+    - Microsoft.Compute/disks/delete
+    - Microsoft.Compute/images/read
+    - Microsoft.Compute/images/write
+    - Microsoft.Compute/locations/diskOperations/read
+    - Microsoft.Compute/skus/read
+    - Microsoft.Compute/virtualMachineScaleSets/delete
+    - Microsoft.Compute/virtualMachineScaleSets/read
+    - Microsoft.Compute/virtualMachineScaleSets/write
+    - Microsoft.Compute/virtualMachines/extensions/write
+    - Microsoft.ContainerService/managedClusters/agentPools/write
+    - Microsoft.ContainerService/managedClusters/delete
+    - Microsoft.ContainerService/managedClusters/write
+    - Microsoft.Network/applicationSecurityGroups/delete
+    - Microsoft.Network/applicationSecurityGroups/read
+    - Microsoft.Network/applicationSecurityGroups/write
+    - Microsoft.Network/bastionHosts/delete
+    - Microsoft.Network/bastionHosts/write
+    - Microsoft.Network/loadBalancers/inboundNatRules/delete
+    - Microsoft.Network/loadBalancers/inboundNatRules/write
+    - Microsoft.Network/natGateways/delete
+    - Microsoft.Network/natGateways/read
+    - Microsoft.Network/natGateways/write
+    - Microsoft.Network/networkInterfaces/delete
+    - Microsoft.Network/networkInterfaces/read
+    - Microsoft.Network/networkInterfaces/write
+    - Microsoft.Network/networkSecurityGroups/delete
+    - Microsoft.Network/networkSecurityGroups/read
+    - Microsoft.Network/networkSecurityGroups/write
+    - Microsoft.Network/privateDnsZones/delete
+    - Microsoft.Network/privateDnsZones/write
+    - Microsoft.Network/privateEndpoints/delete
+    - Microsoft.Network/privateEndpoints/write
+    - Microsoft.Network/publicIPAddresses/delete
+    - Microsoft.Network/publicIPAddresses/read
+    - Microsoft.Network/publicIPAddresses/write
+    - Microsoft.Network/routeTables/delete
+    - Microsoft.Network/routeTables/read
+    - Microsoft.Network/routeTables/write
+    - Microsoft.Network/virtualNetworks/delete
+    - Microsoft.Network/virtualNetworks/delete
+    - Microsoft.Network/virtualNetworks/read
+    - Microsoft.Network/virtualNetworks/subnets/delete
+    - Microsoft.Network/virtualNetworks/subnets/read
+    - Microsoft.Network/virtualNetworks/subnets/write
+    - Microsoft.Network/virtualNetworks/virtualNetworkPeerings/read
+    - Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write
+    - Microsoft.Network/virtualNetworks/write
+    - Microsoft.Resourcehealth/healthevent/action
+    - Microsoft.Resources/subscriptions/resourceGroups/delete
+    - Microsoft.Resources/subscriptions/resourceGroups/read
+    - Microsoft.Resources/subscriptions/resourceGroups/write
+    - Microsoft.Storage/storageAccounts/vmImages/read
```

Comment:
https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-azure-modern/1700048624446083072

Another failure here. Maybe ` Microsoft.ClassicStorage/storageAccounts/read` is the right one?

I am testing with

```workflow-launch openshift-e2e-azure-manual-oidc-workload-identity https://github.com/openshift/cluster-capi-operator/pull/124,https://github.com/openshift/cluster-api-provider-azure/pull/280,https://github.com/openshift/cluster-capi-operator/pull/115 "ENABLE_TECH_PREVIEW_CREDENTIALS_REQUESTS=true"```

Reply:
Changed that, also tried to run the workflow with the command you provided and successfully created a cluster:

https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-azure-modern/1700075503169310720